distribution/docs/src/main/resources/_contents/_catalog-frameworks/attribute-type-definition-contents.adoc

```diff
@@ -0,0 +1,68 @@
+
+==== Attribute Type Definition File
+
+To define Attribute Types, the definition file must have an `attributeTypes` key in the root object.
+
+[source,json]
+----
+{
+    "attributeTypes": {...}
+}
+----
+
+The value of `attributeTypes` must be a map where each key is the attribute type's name and each value is a map that includes the data type and whether the attribute type is stored, indexed, tokenized, or multi-valued.
```

Comment:
Tokenization can also strip things like punctuation.  If tokenization is false then only an exact string will match.